docker/Dockerfile.marimo

```
@@ -172,37 +172,32 @@ EOF
+pixi add --pypi "marimo == $(/usr/local/bin/marimo --version)"
 EOF
-ENV PATH="/home/${USER}/.local/bin:${PATH}"
-
-COPY --chown=${USER} docker/setup/conda-environment.yaml environment.yaml
-COPY --chown=root --chmod=u+rwx,go=rx docker/setup/conda-helper.py /usr/local/bin/conda-helper.py
-ENV CONDA_PREFIX="/home/${USER}/venv"
-ENV CONDA_DEFAULT_ENV="/home/${USER}/venv"
-RUN conda-helper.py sync --verbose
-ENV PATH="/home/${USER}/venv/bin:${PATH}"
-COPY --chown=${USER} docker/setup/conda-pyproject.toml pyproject.toml
+ENV VIRTUAL_ENV="/home/me/.cache/rattler/cache/envs/workspace-5509610833061971390/envs/default"
```

Hadolint (2.15.1)

[error] unexpected 'E'
expecting a new line followed by the next instruction

(DL1000)